### ISSUE TITLE
Google Yubikey U2F support broken

### ISSUE THREAD

**Issue by KenMacD** [gorhill/uMatrix#440]
When attempting to authentication with to a Google account using a Yubikey U2F device the message does not make it to the device unless you authorize all of https://gstatic.com. It's not enough to simply authorize scripts on that domain, you have to mark it green on the left.

How to reproduce (In Chrome):

1. Setup Gmail to use u2f, and register a key
2. Attempt to login, and end up at the u2f page
(The only listed block at the point is scripts on https://www.gstatic.com)
3. Authorize scripts at https://www.gstatic.com
4. Reload

What happen:

No communication between Chrome and Yubikey, can't log in.

What should happen:

Blue light on Yubikey should flash, allowing login.

Workaround:

Authorize the whole https://gstatic.com domain

While a workaround does exist, it isn't ideal as there's no indication that anything was actually blocked on the page.

**Comment by gorhill** [comment]
[Front page of the project](https://github.com/gorhill/uMatrix#warnings):

> **Regarding broken sites**
> 
> uMatrix does not guarantee that sites will work fine: it is for advanced users who can figure how to un-break sites, because essentially uMatrix is a firewall which works in relaxed block-all/allow-exceptionally mode out of the box: it is not unexpected that sites will break.
> 
> **So this means do not file issues to report broken sites when the sites are broken because uMatrix does its job as expected.** I will close any such issue without further comment.

**Comment by KenMacD** [comment]
I read that. The problem is that it didn't "Do it's job as expected". It gave no indication that anything was actually being blocked. The bug isn't "This site breaks", it's that "uMatrix doesn't handle whatever magic Chrome is doing for u2f support"

**Comment by gorhill** [comment]
Can you show me a screenshot of your matrix for when it does not work, prior to allowing whole `gstatic.com` domain?

**Comment by KenMacD** [comment]
Sure. I think I've figured out more of the issue though. It appears there are columns in the plugin that I just can't see. (I've been using this for months and this is the first time I've realized)

In chrome://settings I have the  `Web content` `Page zoom`  set to 200%, because I use a high-dpi screen. If I set it back to to 100% I can see there's a `3` in the XHR column, and if I allow those then the page works as expected.

So I guess it's not a u2f issue, just a UI issue. At 200% and you see:

![ss](https://cloud.githubusercontent.com/assets/8961/11905508/3c98d8a0-a59d-11e5-91b0-95905caa1e8d.png)
